packages/yoga/src/Divider/native/Divider.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import theme from '../../Theme/helpers/themeReader';
```

Comment:
I don't know if it is a pattern but we could avoid getting the theme from `themeReader ` helper by just updating the props from line 6.